woocommerce/woocommerce
# Add filter woocommerce_display_review_rating_form

Added filter woocommerce_display_review_rating_form to single-product-reviews template
This allows us to remove the form via filter

### All Submissions:






### Changes proposed in this Pull Request:
Add a new filter `woocommerce_display_review_rating_form` in the template 'single-product-reviews'

This allows us to remove the review submission form and only display reviews if they are collected in another way.



### How to test the changes in this Pull Request:

1. add `add_filter( 'woocommerce_display_review_rating_form', '__return_false' ); ` in functions.php
2. See form disappear
3. Magic

### Other information:




### Changelog entry
Developer: Added a filter `woocommerce_display_review_rating_form` this could be used to hide the review submission form
> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
